.github/workflows/cicd.yml

```diff
@@ -95,6 +95,11 @@ jobs:
         with:
           lfs: true
 
+      - name: Build assets
+        run: |
+          node build_filter_trees.js
+          node build_templates.js
```

Comment:
I think we should update the publish action instead to run npm run prepublish instead, and update the package.json